core/src/sumeragi/view_change.rs

```diff
@@ -92,54 +98,30 @@ impl Proof {
     }
```

Comment:
I suggested to change this some time ago to just `valid_count > max_faults`, but @appetrosyan  and @SamHSmith preferred it this way, because it's closer to whitepaper formula.